llvm/test/CodeGen/AArch64/lower-ptrmask-arm64_32.ll

```diff
@@ -0,0 +1,13 @@
+; RUN: llc -mtriple=arm64_32-apple-watchos -stop-after=finalize-isel %s -o - | FileCheck %s
```

Comment:
Drop the `-stop-after=finalize-isel` and use update_llc_test_checks.

Reply:
Not sure why, but UTC doesn't produce meaningful CHECK lines on this one. Am I holding it wrong?

```
$ cat /Users/jonathan_roelofs/llvm-upstream/llvm/test/CodeGen/AArch64/lower-ptrmask-arm64_32.ll
; RUN: llc -mtriple=arm64_32-apple-watchos %s -o - | FileCheck %s

define ptr @issue94075(ptr %p) {
entry:
  %rdar125263567 = call ptr @llvm.ptrmask.p0.i32(ptr %p, i32 4294967288)
  ret ptr %rdar125263567
}
$ ../llvm/utils/update_llc_test_checks.py --llc-binary=./bin/llc /Users/jonathan_roelofs/llvm-upstream/llvm/test/CodeGen/AArch64/lower-ptrmask-arm64_32.ll -v
Scanning for RUN lines in test file: /Users/jonathan_roelofs/llvm-upstream/llvm/test/CodeGen/AArch64/lower-ptrmask-arm64_32.ll
Found 1 RUN lines in /Users/jonathan_roelofs/llvm-upstream/llvm/test/CodeGen/AArch64/lower-ptrmask-arm64_32.ll:
  RUN: llc -mtriple=arm64_32-apple-watchos %s -o - | FileCheck %s
Extracted LLC cmd: llc -mtriple=arm64_32-apple-watchos  -o -
Extracted FileCheck prefixes: ['CHECK']
Rewriting FileCheck prefixes: {'CHECK'}
Writing 10 lines to /Users/jonathan_roelofs/llvm-upstream/llvm/test/CodeGen/AArch64/lower-ptrmask-arm64_32.ll...
$ cat /Users/jonathan_roelofs/llvm-upstream/llvm/test/CodeGen/AArch64/lower-ptrmask-arm64_32.ll
; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
; RUN: llc -mtriple=arm64_32-apple-watchos %s -o - | FileCheck %s

define ptr @issue94075(ptr %p) {
entry:
  %rdar125263567 = call ptr @llvm.ptrmask.p0.i32(ptr %p, i32 4294967288)
  ret ptr %rdar125263567
}
;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
; CHECK: {{.*}}
```

Reply:
Probably it doesn't support the arm64_32 triple right now.

Reply:
ohh:
```
def get_run_handler(triple):
    target_handlers = {
        "i686": (scrub_asm_x86, ASM_FUNCTION_X86_RE),
        "x86": (scrub_asm_x86, ASM_FUNCTION_X86_RE),
        "i386": (scrub_asm_x86, ASM_FUNCTION_X86_RE),
        "arm64_32-apple-ios": (scrub_asm_arm_eabi, ASM_FUNCTION_AARCH64_DARWIN_RE),
        "arm64_32-apple-watchos2.0.0": (
            scrub_asm_arm_eabi,
            ASM_FUNCTION_AARCH64_DARWIN_RE,
        ),
 ```